packages/amplify-cli/package.json

```diff
@@ -52,6 +52,7 @@
     "amplify-dotnet-function-runtime-provider": "1.2.0",
     "amplify-dotnet-function-template-provider": "1.3.0",
     "amplify-frontend-android": "2.13.4",
+    "amplify-frontend-flutter": "0.0.1",
```

Comment:
If we start with 0.0.1 the published version will be 0.1.1, not? I think it is ok to start with 1.0.0 here so 1.1.0 will be the published version if you don't know a shortcut to make lerna to make it 1.0.0

Reply:
Doesn't 1.0 suggest that the feature is stable?  This is feature is explicitly in preview.